src/Client/Image/ImageFileUpload.php

```diff
@@ -31,7 +31,22 @@ public function __construct(string $filename)
     {
         $this->validateFile($filename);
         $this->imageFormat = self::ALLOWED_FILE_TYPES[exif_imagetype($filename)];
```

Comment:
Can you add `ext-exif` to composer.json as a required extension? i've missed this.